elasticsearch/install.sh

```diff
@@ -5,11 +5,15 @@ set -ex
 mkdir -p ${HOME}
 ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
 
-/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5.1 -url https://github.com/lukas-vlcek/origin-aggregated-logging/releases/download/v0.1/search-guard-0.5.1.zip
+/usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-ssl/2.3.5.15
```

Comment:
Does it make sense to simplify future changes by moving the versions to env vars?